lib/improver/wxcode/wxcode_utilities.py

```diff
@@ -92,6 +92,15 @@ def add_wxcode_metadata(cube):
     cube.attributes.update({'weather_code': wx_keys})
     wxstring = " ".join(WX_DICT.values())
     cube.attributes.update({'weather_code_meaning': wxstring})
+    # If forecast_period or time have bounds remove them
+    # as the weather code cube is not considered to be over a period.
+    for coord in cube.coords():
+        if coord.name() == 'forecast_period':
```

Comment:
I'd do this in a loop.

Reply:
Is it quicker in a loop?

Reply:
Nope, it can just be a bit cleaner/more pythonic, that's all.
I was thinking something like
```
for coord_name in ['forecast_period', 'time']:                                  
    try:                                                                        
        coord = cube.coord(coord_name)                                          
        if coord.bounds is not None:                                            
            cube.coord(coord).bounds = None                                     
    except iris.exceptions.CoordinateNotFoundError:                             
        continue 
```
that way if there was a mistake in the logic, or you wanted to add another coord or something like that, you wouldn't have to change as much code.

Reply:
Actually that would be quicker than looping over all coords.